Makefile

```diff
@@ -20,8 +20,12 @@ compile-dev:
 	yarn compile:renderer --cache
 
 dev:
+ifeq "$(DETECTED_OS)" "Windows"
+	make init
```

Comment:
TODO: `test` is not valid command in Windows and need to figure out proper way to do this on Windows

Reply:
I second this PR - I had made very similar changes on my fork because I am starting to hack on this project. :-)